READMEs/dispenser-flow.md

```diff
@@ -0,0 +1,126 @@
+<!--
+Copyright 2021 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Dispenser Flow
+
+This quickstart describes dispenser flow.
+
+It focuses on Alice's experience as a publisher
+
+Here are the steps:
+
+1.  Setup
+2.  Alice creates a data token
+3.  Dispenser creation & activation
+
+Let's go through each step.
+
+## 1. Setup
+
+### Prerequisites
+
+-   Linux/MacOS
+-   Docker, [allowing non-root users](https://www.thegeekdiary.com/run-docker-as-a-non-root-user/)
+-   Python 3.8.5+
+
+### Run barge services
+
+In a new console:
+
+```console
+#grab repo
+git clone https://github.com/oceanprotocol/barge
+cd barge
+
+#clean up old containers (to be sure)
+docker system prune -a --volumes
+
+#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh
+```
+
+### Install the ocean.py library
+
+In a new console that we'll call the _work_ console (as we'll use it later):
+
+```console
+#Create your working directory
+mkdir test3
+cd test3
+
+#Initialize virtual environment and activate it.
+python -m venv venv
+source venv/bin/activate
+
+#Install the ocean.py library. Install wheel first to avoid errors.
+pip install wheel
+pip install ocean-lib
+```
+
+### Set envvars
+
+In the work console:
+```console
+#set private keys of two accounts
+export TEST_PRIVATE_KEY1=0xbbfbee4961061d506ffbb11dfea64eba16355cbf1d9c29613126ba7fec0aed5d
+
+#set network URL
+export OCEAN_NETWORK_URL=http://127.0.0.1:8545
+
+#start python
+python
+```
+
+## 2. Alice creates the data token
+
+
+In the Python console:
+```python
+#Create ocean instance
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.ocean.ocean import Ocean
+config = ExampleConfig.get_config()
+ocean = Ocean(config)
+
+print(f"config.network_url = '{config.network_url}'")
+print(f"config.metadata_cache_uri = '{config.metadata_cache_uri}'")
+print(f"config.provider_url = '{config.provider_url}'")
+
+#Alice's wallet
+import os
+from ocean_lib.web3_internal.wallet import Wallet
+alice_wallet = Wallet(ocean.web3, private_key=os.getenv('TEST_PRIVATE_KEY1'))
+print(f"alice_wallet.address = '{alice_wallet.address}'")
+assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+data_token = ocean.create_data_token('DataToken1', 'DT1', alice_wallet, blob=ocean.config.metadata_cache_uri)
```

Comment:
I would add an empty line before the data token instructions, to better separate the wallet and data token parts.